src/server/world.zig

```diff
@@ -818,6 +818,10 @@ pub const ServerWorld = struct { // MARK: ServerWorld
 			if (main.items.Inventory.Sync.ServerSide.getInventoryFromSource(.playerInventory)) |inv| {
 				playerZon.put("inventory", inv.save(main.stackAllocator));
 			}
+
+			if (main.items.Inventory.Sync.ServerSide.getInventoryFromUserAndSource(user, .other)) |inv| {
```

Comment:
.other could also return you the crafting table or whatever. You are just lucky here that the hand is stored first in the list.